xyz-psql-connector/src/main/java/com/here/xyz/psql/PSQLXyzConnector.java

```diff
@@ -874,21 +874,22 @@ protected XyzResponse processModifyFeaturesEvent(ModifyFeaturesEvent event) thro
     }
 
     final long now = System.currentTimeMillis();
+    final boolean addUUID = event.getEnableUUID() && event.getVersion().compareTo("0.2.0") < 0;
```

Comment:
addUUID should't be when the event's version is equals or greater than 0.2.0 instead of lower than?
in this case: event.getVersion().compareTo("0.2.0") >= 0'

Reply:
The check there is correct. The connector should add the UUID only for versions smaller than 0.2.0.
For versions 0.2.0 and higher, the service will handle the UUID values. This is handled in https://github.com/heremaps/xyz-hub/blob/79f518974e6d8c9d2cf88d7504943806fdc49f23/xyz-hub-service/src/main/java/com/here/xyz/hub/task/FeatureTaskHandler.java#L696